### PR TITLE
[GITHUB-10] Add suport for systemd

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
Systemd is now the default service management on Ubuntu Xenial. This
commit adds support for both init and systemd.

It also add support for using different Vagrant images. Simply edit
`.make` file and add this line

```
VAGRANT_BOX=ubuntu/xenial64
```

If you want to test installation of a differen versions of Kibana, edit
`.make` file and add

```
KIBANA_VARS=5.0
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,17 @@
+include .make
+VAGRANT_BOX ?= ubuntu/trusty64
+KIBANA_VARS ?=
```
Comment (dm) [inline Makefile:3]: Maybe set the default to `5.0`?

Comment (wojtek-oledzki) [reply Makefile:3]: defaults are in `tests/vars_kibana_default.yml`

**Comment by lobsterdore** [comment]
Testing this now.

**Comment by dm** [comment]
@lobsterdore WFM 😄 👍

**Comment by lobsterdore** [comment]
OK so this all works for, but can you add a travis check for Kb 4.1 as well? We need to maintain support for 4.1 for a project.

**Comment by wojtek-oledzki** [comment]
@lobsterdore what about:
1. update defaults/main.yml to 4.6.2
2. add teste vars for 4.1.2

would that work for you?

**Comment by lobsterdore** [comment]
LGTM :+1:

**Pull request merged**